dependencies.gradle

```diff
@@ -0,0 +1,23 @@
+ext {
+    versions = [
+            androidSdk    : [
+                    compile   : 27,
+                    min       : 23,
+                    target    : 27,
+                    buildTools: '26.0.2'
+            ],
+            androidSupport: '27.0.0',
+            kotlin        : '1.1.51'
+    ]
+
+    libraries = [
+            build             : [
+                    androidGradle: 'com.android.tools.build:gradle:3.0.1',
+                    kotlinGradle : "org.jetbrains.kotlin:kotlin-gradle-plugin:${versions.kotlin}"
```

Comment:
here's an example of where we use double quotes because we're using the `${fooooo}` thing to reference a variable. the kotlin version is used in multiple places so we extract it.